systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAuthorizationIsolatedST.java

```diff
@@ -686,6 +686,7 @@ void setUp(ExtensionContext extensionContext)  {
     @AfterAll
     void tearDown(ExtensionContext extensionContext) throws Exception {
         // delete keycloak before namespace
+        KeycloakUtils.deletePostgres(Constants.INFRA_NAMESPACE);
```

Comment:
Guess it would be better to delete it after keycloak, not before

Reply:
Yeah, good spot. I will change order as you says. Apparently it works fine even this way. :-)